app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  validates :name, presence: true
+  validates :photo, presence: true
+  validates :bio, presence: true
+
+  def recent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
+
+  def update_posts_counter
+    update(posts_counter: posts.count)
+  end
```

Comment:
- Hello, I would advise you to move this method(update_pots_counter) to `post.rb`  because the requirement says that post model should have a method that updates the posts counter for a user